frontend/src/api/network/servingRuntimes.ts

```diff
@@ -11,80 +11,72 @@ import { ServingRuntimeKind } from '../../k8sTypes';
 import { CreatingServingRuntimeObject } from 'pages/modelServing/screens/types';
 import { getModelServingRuntimeName } from 'pages/modelServing/utils';
 import { getModelServingProjects } from './projects';
+import { getConfigMap } from './configMaps';
+import { DEFAULT_MODEL_SERVING_TEAMPLATE } from 'pages/modelServing/screens/const';
+import YAML from 'yaml';
+
+const fetchServingRuntime = (
+  data: CreatingServingRuntimeObject,
+  configNamespace: string,
+  namespace: string,
+): Promise<ServingRuntimeKind> => {
+  return getConfigMap(configNamespace, 'servingruntimes-config')
+    .then((configmap) => {
+      const servingRuntime = YAML.parse(configmap.data?.['default-config'] || '');
+      if (servingRuntime === null) {
+        throw new Error(
+          'servingruntimes-config is misconfigured or key might be missing from the ConfigMap',
+        );
+      }
+      return assembleServingRuntime(data, namespace, servingRuntime);
+    })
+    .catch((e) => {
+      console.log(`${e}, using default config.`);
```

Comment:
Only concern I have after testing this back and forth is -- this is going to be hard to see as an error in prod -- any concerns about making it a `console.error`? I don't think it should cause any issues.